### PR TITLE
Change iff to if and only if to improve annotation readability

### PR DESCRIPTION
**Pull request by liguangbo**
The `iff` is the abbreviation for `if and only if`, iff is not friendly for some people.
Change `iff` to `if and only if` to improve annotation readability.

### CODE REVIEW

**Comment by jlowdermilk** [comment]
I'm not strictly opposed to the change, but I'm curious why you think `iff` is unfriendly? It's a familiar abbreviation for programming/logic systems, and the occurrences are all in comments/docs that target kubernetes devs, not end users.

**Comment by liguangbo** [comment]
@jlowdermilk There is also some if and only if in the codes. The `iff` and `if and only if` exist the same time. Someone not good in English may misunderstand.

File: hack/update-generated-swagger-docs.sh [hunk 1]
```diff
@@ -37,7 +37,7 @@ function generate_version() {
 // generate Swagger API documentation for its models. Please read this PR for more
 // information on the implementation: https://github.com/emicklei/go-restful/pull/215
 //
-// TODOs are ignored from the parser. (e.g. TODO(andronat):... || TODO:...) iff
+// TODOs are ignored from the parser. (e.g. TODO(andronat):... || TODO:...) if and only if
 // are on one line! For multiple line or blocks that you want to ignore use ---.
```
Comment (jlowdermilk) [inline hack/update-generated-swagger-docs.sh]: add "they" (if and only if they are...)

File: hack/update-generated-swagger-docs.sh [hunk 2]
```diff
@@ -37,7 +37,7 @@ function generate_version() {
 // generate Swagger API documentation for its models. Please read this PR for more
 // information on the implementation: https://github.com/emicklei/go-restful/pull/215
 //
-// TODOs are ignored from the parser. (e.g. TODO(andronat):... || TODO:...) iff
```
Comment (jlowdermilk) [inline hack/update-generated-swagger-docs.sh]: Remove `.` after parser. Reads better as one sentence.

**Comment by jlowdermilk** [comment]
> @jlowdermilk There is also some if and only if in the codes. The iff and if and only if exist the same time. Someone not good in English may misunderstand.

Fair enough, expanding for clarity sounds fine to me. Couple nits, otherwise lgtm

**Comment by liguangbo** [comment]
@jlowdermilk Thanks for your guidance.

**Comment by liguangbo** [comment]
shippable time out

**Comment by liguangbo** [comment]
shippable time out

**Comment by liguangbo** [comment]
I can't view the shippable.
image
Someone can help to view it?

**Comment by jlowdermilk** [comment]
Shippable seems to be having environment issues. Build restarted.

**Comment by liguangbo** [comment]
@jlowdermilk Thank you very much.

**Comment by eparis** [comment]
hopefully the little magic robot will come in while I'm sleeping and this will merge.

**Comment by liguangbo** [comment]
@eparis I think I need another lgtm. The little magic robot removed lgtm label, but I didn't change the code. Thank you. ;-)

**Pull request merged**